frontend/assessments/1/style.scss

```diff
@@ -0,0 +1,153 @@
+@import "variables.scss";
+body{
+    background-color: $background-color;
+    padding-left: 2rem;
+    padding-right: 2rem;
+    color:$primary-text-color;
+    font-family: "Poppins";
+}
+
+.navbar{
+    margin: 2rem;
+    font-weight: 650;
+
+    display: flex;
+    flex: 1 3 ;
+    justify-content: space-between;
+    align-items: center;
+    .right-navbar{
+    //   margin-right: 5rem;
+        display: flex;
+        div{
+            margin-left: 4rem;
+        }
+    }
+}
+
+.main-container
+{
+    margin-top: 4rem;
+    display: flex;
+    flex: 1 1;
+    justify-content: space-evenly;
+    .big-text
+    {
+        font-size: 6rem;
+        font-weight: 1000;
+    }
+    p{
+        margin-top: 0;
+        margin-bottom: 0;
+    }
+    img{
+         height: 400px;
+        width:fit-content
+    }
+    .sub-text
+    {
+        font-weight: bold;
+    }
+   
+}
+.hamburger{
+    display: none;
+}
+.btn
+{
+    margin-top: 1rem;
+    background-color: $primary-button-color; /* Green */
+    border: none;
+    color: white;
+    padding: 1rem 1rem;
+    text-align: center;
+    text-decoration: none;
+    display: inline-block;
+    font-size: 16px;
+    font-weight: bold;
+    border-radius: .5rem;
+}
+@media screen and (max-width: 576px) {
+    .main-container {
+        flex-direction: column-reverse;
+
+        .big-text {
+            font-size: 60px;
+        }
+
+        img {
+            height: 300px;
+        }
+    }
+
+
+  
+    .hamburger:hover + div {
+        
+            display: flex;
+            flex-direction: column;
+            justify-content: space-between;
+            margin-top: 20px;
+            max-height: 500px; 
+            transition: max-height 0.5s ease;
+        p{
+            display: block;
+            margin: 0;
+        }
+        
+    }
+
+    .navbar{
+      
+        p{
+            display: none;
+        }
+        img
+        {
+            position: absolute;
+            right: 20px;
+        }
+        img:hover + div{
+            display: block;
+        }
+    }
+    .hamburger {
+        display: block;
+        height: 40px;
+    }
+
+   
+    .hamburger:hover  .right-navbar {
+        display: flex;
+        flex-direction: column;
+        justify-content: space-between;
+        height: auto;
+        max-height: 500px; 
+        transition: max-height 0.5s ease; /* Smooth transition effect */
+    }
+}
+
+@media screen and  (min-width:577px)  and (max-width:1000px){
+   
+    .main-container {
+        padding-left: 20px;
+        margin-top: 7rem;
+    }
+    .main-container .big-text {
+        font-size: 70px;
+    }
+    .main-container img {
+        height: 200px;
+      
+    }
+    p.big-text {
+      margin-left: 10px;
```

Comment:
lets try to use rem/em for properties and add common values in the variables in the scss file.